lib/export/jdl_exporter.js

```diff
@@ -18,11 +18,155 @@
  */
 
 const fs = require('fs');
+const path = require('path');
+const applicationOptions = require('../core/jhipster/application_options');
+const unaryOptions = require('../core/jhipster/unary_options');
+const JSONFileReader = require('../reader/json_file_reader');
+const JSONReader = require('../reader/json_reader');
+const JSONToJdlApplicationConverter = require('../converters/json_to_jdl_application_converter');
+const FileUtils = require('../utils/file_utils');
+const JDLObject = require('../core/jdl_object');
 
 module.exports = {
-  exportToJDL
+  exportToJDL,
+  loadJhipsterConfigAndEntities
 };
 
+const excludeUnaryOptions = ['skipUserManagement'];
```

Comment:
I initially thought it was a function, not a variable. Did you mean `excludedUnaryOptions`?

Reply:
And a Set would be better

Reply:
Yes, that is what I mean, some .yo-rc.json options or default options in the JDLApplication are not supported in the jdl file.